### ISSUE TITLE
fix what roxygen broke 🙄

### ISSUE THREAD

**Issue by brunj7** [NCEAS/metajam#141]
---------- Forwarded message ---------
From: Kurt Hornik <[Kurt.Hornik@wu.ac.at](mailto:Kurt.Hornik@wu.ac.at)>
Date: Sat, Aug 19, 2023 at 01:49
Subject: CRAN package metajam
To: <[brun@nceas.ucsb.edu](mailto:brun@nceas.ucsb.edu)>
CC: <[Kurt.Hornik@r-project.org](mailto:Kurt.Hornik@r-project.org)>

Dear maintainer,

You have file 'metajam/man/metajam.Rd' with \docType{package}, likely
intended as a package overview help file, but without the appropriate
PKGNAME-package \alias as per "Documenting packages" in R-exts.

This seems to be the consequence of the breaking change

  Using @docType package no longer automatically adds a -package alias.
  Instead document _PACKAGE to get all the defaults for package
  documentation.

in roxygen2 7.0.0 (2019-11-12) having gone unnoticed, see
<https://github.com/r-lib/roxygen2/issues/1491>.

As explained in the issue, to get the desired PKGNAME-package \alias
back, you should either change to the new approach and document the new
special sentinel

  "_PACKAGE"

or manually add

  @aliases metajam-package

if remaining with the old approach.

Please fix in your master sources as appropriate, and submit a fixed
version of your package within the next few months.

Best,
-k

**Comment by njlyon0** [comment]
May be as simple a fix as adding "_PACKAGE" to [metajam-package.R](https://github.com/NCEAS/metajam/blob/main/R/metajam-package.R) or there may be a nice `usethis` function for doing it automatically

**Comment by brunj7** [comment]
https://usethis.r-lib.org/reference/use_package_doc.html

**Comment by njlyon0** [comment]
I _think_ I fixed this. Hard to know for sure until we submit to CRAN again and/or the CRAN reviewers checked / confirmed this. That said, I did use the `usethis::use_package_doc` and the output did make `metajam-package.R` look exactly like its equivalents in my other R packages (which haven't gotten this piece of feedback from reviewers) so I think we can safely close this for the sake of the milestone's tracking.

Worst case we can always re-open this issue if needed.
